samples/word/35-ranges/get-pages.yaml

```
@@ -0,0 +1,244 @@
+        $("#get-pages-third-paragraph").on("click", () => tryCatch(getPagesOfThirdParagraph));
+        $("#get-pages-enclosing-viewport").on("click", () => tryCatch(getPagesEnclosingViewport));
+        $("#get-all-pages").on("click", () => tryCatch(getAllPages));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function getPagesOfSelectedRange() {
+          await Word.run(async (context) => {
+            // Gets pages of the selection.
+            const pages: Word.PageCollection = context.document.getSelection().pages;
+            pages.load();
+            await context.sync();
+
+            console.log(pages);
+            // Log info for pages included in selection.
+            for (let i = 0; i < pages.items.length; i++) {
+              let page = pages.items[i];
+              page.load();
+              await context.sync();
+
+              console.log(`Index info for page ${i + 1} in the selection: ${page.index}`);
+              const range = page.getRange();
+              range.load();
+              await context.sync();
+
+              console.log("Text of that page in the selection:", range.text);
+            }
+          });
+        }
+
+        async function getPagesOfThirdParagraph() {
+          await Word.run(async (context) => {
+            // Gets the pages that the third paragraph is found on.
+            const paragraphs: Word.ParagraphCollection = context.document.body.paragraphs;
+            paragraphs.load();
+            await context.sync();
+
+            const paraThree = paragraphs.items[2];
+            const rangeOfParagraph = paraThree.getRange();
+            await context.sync();
+
+            const pages: Word.PageCollection = rangeOfParagraph.pages;
+            pages.load();
+            await context.sync();
+
+            console.log(pages);
+            // Log into for pages in range.
+            for (let i = 0; i < pages.items.length; i++) {
+              let page = pages.items[i];
+              page.load();
+              await context.sync();
+
+              console.log(`Index of page ${i + 1} that the third paragraph is found on: ${page.index}`);
+              const range = page.getRange();
+              range.load();
+              await context.sync();
+
+              console.log("Text of that page:", range.text);
+            }
+          });
+        }
+
+        async function getPagesEnclosingViewport() {
+          await Word.run(async (context) => {
+            // Gets the pages enclosing the viewport.
+
+            // Get the active window.
+            let activeWindow = context.document.activeWindow;
+            activeWindow.load();
+            await context.sync();
+
+            // Get the active pane.
+            let activePane = activeWindow.activePane;
+            activePane.load();
+            await context.sync();
+            // Get pages enclosing the viewport.
```

I find adding a blank line after syncs helps with readability.
```suggestion
            await context.sync();
            
            // Get pages enclosing the viewport.
```